docs/walkthrough/updates.rst

```diff
@@ -28,17 +23,17 @@ with one additional line:
 .. code-block:: python
     :linenos:
     :caption: ephemeral.py
-    :emphasize-lines: 23
+    :emphasize-lines: 24
 
     @kopf.on.create('zalando.org', 'v1', 'ephemeralvolumeclaims')
-    def create_fn(spec, meta, namespace, logger, **kwargs):
+    def create_fn(body, spec, meta, namespace, logger, **kwargs):
```

Comment:
don't need `body` here.  delete `kopf.adopt(data, owner=body)` because you only show it later in `cascaded deletion`.